client/src/crane.ts

```diff
@@ -56,6 +56,10 @@ export default class Crane
         Crane.statusBarItem.tooltip = "Crane is processing the PHP source files in your workspace to build code completion suggestions";
         Crane.statusBarItem.show();
 
+        var statusBarItem: StatusBarItem = window.createStatusBarItem(StatusBarAlignment.Right);
+        statusBarItem.text = Config.version;
+        statusBarItem.show();
```

Comment:
Can we add a hover title to say something like "Crane (PHP Code-completion) version <x>" ? (Much like the Typescript one does)